lms/djangoapps/courseware/tests/test_transformers.py

```diff
@@ -0,0 +1,250 @@
+"""
+Test the behavior of the GradesTransformer
+"""
+
+import datetime
+import pytz
+import random
+
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import check_mongo_calls
+
+from lms.djangoapps.course_blocks.api import _get_cache
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from lms.djangoapps.course_blocks.transformers.tests.helpers import CourseStructureTestCase
+from ..transformers.grades import GradesTransformer
+
+
+class GradesTransformerTestCase(CourseStructureTestCase):
+    """
+    Verify behavior of the GradesTransformer
+    """
+
+    TRANSFORMER_CLASS_TO_TEST = GradesTransformer
+
+    problem_metadata = {
+        u'graded': True,
+        u'weight': 1,
+        u'due': datetime.datetime(2099, 3, 15, 12, 30, 0, tzinfo=pytz.utc),
```

Comment:
Nit: Instead of hard-coding a date, you can simply do `datetime.datetime.now() + datetime.timedelta(days=1)`